Documentation/ExtensionArchitecture/ComponentsOfTYPO3Extension/CreatingDatabaseModel.rst

```diff
@@ -0,0 +1,36 @@
+..  include:: /Includes.rst.txt
+.. _creating-database-model:
+
+=============================
+Creating a new database model
+=============================
+
+Create SQL database schema
+---------------------------
+
+* Add `example-extension/ext_tables.sql`
+* Insert your SQL database schema definition into that file
+  * Each entity is represented by one database table
+    * Table name has the following structure: `tx_{extension key without underscores}_domain_model_{entity name}`
+    * Each entity property is represented by one database column
+
+.. code-block:: sql
```

Comment:
For most field types we don't need to add them manually to the ext_tables.sql anymore in TYPO3 13. So I would say: Model the database scheme from the TCA or php model perspective, then check which fields still have to be added to the ext_tables regarding TYPO3 version?

Reply:
That's a very good point, yes. There's really no great workflow for this. Maybe point to the "Auto-db create field" changelog? There we have a documentation which TCA types are auto-created. Or did we migrat that to a documentation page already, to indicate which types are autocreated?! Better to link there than to suggest a workflow like "check which fields are missing in your database, because you'll get errors for some fields but not for others", that might be very frustrating?

Reply:
Let me know if current version works for you.